.travis.yml

```diff
@@ -14,22 +14,12 @@ install:
   - python setup.py -q develop
   - pip install -r requirements-test.txt
 
-# command to run flake8
+# command to run qa-checks
```

Comment:
Comment not required now! :smile:

I think qa-check can move to `script`, so `before_script` is also not required now! :smile:
That'll be cleaner! :smile:

Reply:
Yes, not needed, but I'll merge anyway to speed this up